### PR TITLE
added the navbar

### PR DESCRIPTION
**Pull request by manikorimilli**
# Hey satish i added the navbar functionality to your  code base

- home
- about
- -contact

### CODE REVIEW

**Comment by Satish1321** [comment]
Hye could you please add the cart section in navbar section

**Comment by manikorimilli** [comment]
yes i do in 2 days of time

**Comment by manikorimilli** [comment]
I added the cart section also see

File: navbar.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<nav>
+    <ul>
+        <li>Home</li>
+        <li>About</li>
+        <li>Contact</li>
+        <li>Cart</li>
```
Comment (Satish1321) [inline navbar.html:6]: i really liked this

Comment (manikorimilli) [reply navbar.html:6]: ok thankyou

**Pull request merged**